api/services/orders/order_handler.go

```diff
@@ -0,0 +1,70 @@
+package orders
+
+import (
+	"net/http"
+	"sort"
+
+	"github.com/gorilla/mux"
+	"github.com/sockify/sockify/middleware"
+	"github.com/sockify/sockify/types"
+	"github.com/sockify/sockify/utils"
+)
+
+type Handler struct {
+	orderStore types.OrderStore
+	adminStore types.AdminStore
+}
+
+func NewHandler(orderStore types.OrderStore, adminStore types.AdminStore) *Handler {
+	return &Handler{orderStore: orderStore, adminStore: adminStore}
+}
+
+func (h *Handler) RegisterRoutes(router *mux.Router) {
+	router.HandleFunc("/orders", middleware.WithJWTAuth(h.adminStore, h.handleGetOrders)).Methods(http.MethodGet)
+
+}
+
+// Get all orders, possibly filtered by status
```

Comment:
Add a Swagger doc comment (see the `sock_handler.go` for reference) and regenerate the docs: `make swag` within `api` folder